content/ja/docs/languages/java/getting-started.md

```diff
@@ -0,0 +1,249 @@
+io.opentelemetry.exporter.logging.LoggingSpanExporter - 'RollController.index' :
+ 70c2f04ec863a956e9af975ba0d983ee 7fd145f5cda13625 INTERNAL [tracer:
+ io.opentelemetry.spring-webmvc-6.0:1.25.0-alpha] AttributesMap{data=
+ {thread.id=39, thread.name=http-nio-8080-exec-1}, capacity=128,
+ totalAddedValues=2}
+[otel.javaagent 2023-04-24 17:33:54:568 +0200] [http-nio-8080-exec-1] INFO
+io.opentelemetry.exporter.logging.LoggingSpanExporter - 'GET /rolldice' :
+70c2f04ec863a956e9af975ba0d983ee 647ad186ad53eccf SERVER [tracer:
+io.opentelemetry.tomcat-10.0:1.25.0-alpha] AttributesMap{
+  data={user_agent.original=curl/7.87.0, net.host.name=localhost,
+  net.transport=ip_tcp, http.target=/rolldice, net.sock.peer.addr=127.0.0.1,
+  thread.name=http-nio-8080-exec-1, net.sock.peer.port=53422,
+  http.route=/rolldice, net.sock.host.addr=127.0.0.1, thread.id=39,
+  net.protocol.name=http, http.status_code=200, http.scheme=http,
+  net.protocol.version=1.1, http.response_content_length=1,
+  net.host.port=8080, http.method=GET}, capacity=128, totalAddedValues=17}
+```
+
+ステップ5で、サーバーを停止するときに、収集されたすべてのメトリクスの出力が表示されるはずです（メトリクス出力は便宜上改行および短縮されています）。
+
+```sh
+[otel.javaagent 2023-04-24 17:34:25:347 +0200] [PeriodicMetricReader-1] INFO
+io.opentelemetry.exporter.logging.LoggingMetricExporter - Received a collection
+ of 19 metrics for export.
+[otel.javaagent 2023-04-24 17:34:25:347 +0200] [PeriodicMetricReader-1] INFO
+io.opentelemetry.exporter.logging.LoggingMetricExporter - metric:
+ImmutableMetricData{resource=Resource{schemaUrl=
+https://opentelemetry.io/schemas/1.19.0, attributes={host.arch="aarch64",
+host.name="OPENTELEMETRY", os.description="Mac OS X 13.3.1", os.type="darwin",
+process.command_args=[/bin/java, -jar, java-simple.jar],
+process.executable.path="/bin/java", process.pid=64497,
+process.runtime.description="Homebrew OpenJDK 64-Bit Server VM 20",
+process.runtime.name="OpenJDK Runtime Environment",
+process.runtime.version="20", service.name="java-simple",
+telemetry.auto.version="1.25.0", telemetry.sdk.language="java",
+telemetry.sdk.name="opentelemetry", telemetry.sdk.version="1.25.0"}},
+instrumentationScopeInfo=InstrumentationScopeInfo{name=io.opentelemetry.runtime-metrics,
+version=1.25.0, schemaUrl=null, attributes={}},
+name=process.runtime.jvm.buffer.limit, description=Total capacity of the buffers
+in this pool, unit=By, type=LONG_SUM, data=ImmutableSumData{points=
+[ImmutableLongPointData{startEpochNanos=1682350405319221000,
+epochNanos=1682350465326752000, attributes=
+{pool="mapped - 'non-volatile memory'"}, value=0, exemplars=[]},
+ImmutableLongPointData{startEpochNanos=1682350405319221000,
+epochNanos=1682350465326752000, attributes={pool="mapped"},
+value=0, exemplars=[]},
+ImmutableLongPointData{startEpochNanos=1682350405319221000,
+epochNanos=1682350465326752000, attributes={pool="direct"},
+value=8192, exemplars=[]}], monotonic=false, aggregationTemporality=CUMULATIVE}}
+...
+```
+
+## 次のステップ
```

Comment:
Heading ID needed here.

Reply:
@chalin 

Sorry, I fixed it in a bellow commit.
Please confirm it.

https://github.com/open-telemetry/opentelemetry.io/pull/7057/commits/c49675fa1087aa6937c38e1d7213bd493976903f